projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.ts

```diff
@@ -30,7 +30,18 @@ import { UserPreferencesService } from '@alfresco/adf-core';
 import { DocumentListPresetRef } from '@alfresco/adf-extensions';
 
 @Component({
-  templateUrl: './favorite-libraries.component.html'
+  templateUrl: './favorite-libraries.component.html',
+  styles: [
+    `
+      .sidebar {
```

Comment:
Can we declare this class 1 time and reuse it for all the components that need it?

Reply:
done